content/apidokumentasjon/reelle-rettighetshavere/maskinell-innrapportering/beskrivelse-av-felter/_index.md

```diff
@@ -4,6 +4,66 @@ description: Tekstlig beskrivelse av feltene i JSON-skjemaet
+|                                                                        | registreringspliktigVirksomhet.organisasjonsnummer                                                                                                  | Organisasjonsnummer for den registreringspliktige virksomheten.  Dette vil være den virksomheten som sluttbruker rapporterer for. Feltet blir preutfylt automatisk, og skal aldri endres av fagsystem                                                                                                                                                                                                                                                                                                                                       | Nei         | Ja       |                                                                                                                                                                                                                                                                   |
+|                                                                        | reelleRettighetshavereidentifikasjon                                                                                                                | Kodeverdi (fra kodelista reelleRettighetshavereidentifikasjon) som angir om virksomheten har / kan identifisere reelle rettighetshavere. Angi en av 3 følgende verdier:  1. Har reelle rettighetshavere `reelleRettighetshavereidentifikasjon.harReelleRettighetshavere`  2. Har reelle rettighetshavere, men ingen av disse kan identifiseres `reelleRettighetshavereidentifikasjon.ingenReelleRettighetshavereKanIdentifiseres`  3. Har ikke reelle rettighetshavere `reelleRettighetshavereidentifikasjon.harIkkeReelleRettighetshavere` | Ja          | Ja       |                                                                                                                                                                                                                                                                   |
```

Comment:
Beskrivelsen for reelleRettighetsahavereIdentifikasjon er ikke helt lett å lese. Er det mulig å få punkt 1, 2 og 3 på hver sin linje?